schemas.py

```diff
@@ -0,0 +1,56 @@
+from pydantic import BaseModel
+from datetime import date, datetime
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str | None = None
+    publication_date: date
+
+    class Config:
+        orm_mode = True
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str | None = None
+
+    class Config:
+        orm_mode = True
+
```

Comment:
is this class needed for all schemas that inherit from this one? If not - it is better to provide this option on demand. By using `orm_mode=True` pydantic will validate your data with a less efficient algorithm. You will not mention performance diff on such a simple API, but it is good to know:)

Reply:
Thank you, noted! Indeed good to know, is it allright in such cases to added orm_mode=True for both read and write schemas, i mean it wont be SRP violation right? :)